jib-core/src/main/java/com/google/cloud/tools/jib/api/JibContainerBuilder.java

```diff
@@ -319,7 +316,7 @@ public JibContainerBuilder addLabel(String key, String value) {
    * @return this
    */
   public JibContainerBuilder setFormat(ImageFormat imageFormat) {
-    this.imageFormat = imageFormat;
+    buildConfigurationBuilder.setTargetFormat(imageFormat.getManifestTemplateClass());
```

Comment:
We should probably change the `setTargetFormat` to just take the `ImageFormat` rather than its manifesttemplateclass, but can leave that to later.